File: app/src/main/java/com/theappsolutions/boilerplate/ui/auth/AuthView.java
package com.theappsolutions.boilerplate.ui.auth;

import com.theappsolutions.boilerplate.ui.base.BaseView;

/**
 * @author Severyn Parkhomenko s.parkhomenko@theappsolutions.com
 * @copyright (c) 2018 TheAppSolutions. (https://theappsolutions.com)
 */
public interface AuthView extends BaseView {

    void navigateToMenuScreen();

    void navigateToListScreen();

    void showLoginError(String text);

    void showPasswordError(String text);

}

File: app/src/main/java/com/theappsolutions/boilerplate/ui/auth/AuthView.kt
package com.theappsolutions.boilerplate.ui.auth

import com.theappsolutions.boilerplate.ui.base.BaseView

/**
 * @author Severyn Parkhomenko s.parkhomenko@theappsolutions.com
 * @copyright (c) 2018 TheAppSolutions. (https://theappsolutions.com)
 */
interface AuthView : BaseView {

    fun navigateToMenuScreen()

    fun navigateToListScreen()

    fun showLoginError(text: String)

    fun showPasswordError(text: String)

}